app/code/Magento/InventoryImportExport/Plugin/Import/SourceItemImporter.php

```diff
@@ -1,20 +1,22 @@
 <?php
 /**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
+ * Created by PhpStorm.
+ * User: josh
+ * Date: 05/02/2018
+ * Time: 00:57
```

Comment:
Please fix the Copyright

Reply:
@maghamed I've corrected this now in latest changes